source/_integrations/thread.markdown

```diff
@@ -14,30 +14,158 @@ ha_platforms:
+
+### About different Thread networks
+
+Today, each vendor forms their own Thread network when you start using their products. This means you can end up having a Home Assistant, an Apple, and a Google Thread network in your home. The Thread configuration panel lists all Thread border routers and groups them by the Thread network.
+
+<p class='img'><img width="400" src='/images/integrations/thread/thread-no-3rd-party-credentials.png'>
+The Thread configuration page shows three vendor-specific Thread networks.
+</p>
+
+These are all separate networks using different credentials. This means devices can't roam between the Thread networks.
+
+The <img width="30px" src='/images/integrations/thread/information-outline.png'> icon indicates that Home Assistant has the credentials for that network. In this case, only the credentials of the `home assistant` network are known.
+
+Home Assistant discovers all Thread border routers in your network because they send mDNS/DNS-SD announcements. These local announcements don't contain the network credentials. That's why you see the network there, but not the credentials.
+
+#### About the preferred network
+
+The intention of the **Preferred network** in Home Assistant is that it will be used as the default network when adding Thread⁻based devices.
+
+<div class="note">
+
+The **preferred network** function isn't completely implemented yet. In particular, when adding Matter devices through the companion apps, the preferred network of the mobile device is being used.
+
+</div>
+
+#### Making a network your preferred network
+
+You can only set a Thread network as preferred if the credentials are known.
+
+1. To import Thread credentials, you need your Android and iOS companion app.
+2. On your companion app, navigate to the Thread configuration page.
+   - You should see an **Import credentials** button on the lower right corner.
+
+   <img width="400" src='/images/integrations/thread/thread-import-credentials.png'>
+
+   - Importing the credentials allows a Google- or Apple-created Thread network to be the preferred network of Home Assistant.
+   <img width="400" src='/images/integrations/thread/thread-preferred-network.png'>
+
+#### Using Home Assistant as a <abbr title="Thread border routers">TBRs</abbr> without setting up another vendor's TBR first
+
+This scenario currently only works in one particular case, under the following conditions:
+
+- You do not have a Google <abbr title="Thread border routers">TBRs</abbr> set up yet (otherwise the Google <abbr title="Thread border routers">TBRs</abbr> will be the preferred BR, and Android APIs do not allow to change that currently)
+- You have the OpenThread Border Router add-on installed (or the Silicon Labs Multiprotocol add-on, which includes an OpenThread Border Router). For instructions on how to enable Thread on SkyConnect or Yellow, refer to the following guides:
```

Comment:
please remove the referral to multiprotocol here

Reply:
Iit does not refer to multiprotocol here, it just notes that this is one of the ways to operate OTBR. It is in brackets and it is one of the valid options. We have 7k+ users runnning this option. I'd prefer if we leave it here.

Reply:
I don't agree, we should remove all referrals to multi-pan. It is unstable and causing a shitload of support issues and confused users. I'm fine with people experimenting with it but lets not promote it anywhere, period.

Reply:
This is not promoting. It isn't even a link. 

Removing it from here would hide a fact that Silicon Labs Multiprotocol includes an OpenThread Border Router. 

What if one of this 7k people is wondering where his use case is? They will land on the support, asking "how can I use the multiprotocol I have running and working well on my end?". So if anything, I'd argue hiding the fact that Multiprotocol exists will add support.

I am 100% agree, we should not promote it. But this is not promotion.

Reply:
Even though it adds confusion, just scrap it here.

Reply:
In the border router section, it is mentioned the Multiprotocol add-on includes an OTBR.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/8168c660-646d-4a30-a149-9e0796d4f397)

Would this be sufficient to cover the use case?

Reply:
Actually, it seems a good point to actively promote the dediciated Thread over Multiprotocol. New suggestion: https://github.com/home-assistant/home-assistant.io/pull/30913/files#r1461693490